api/v1alpha1/worker_types.go

```diff
@@ -312,15 +334,21 @@ type ManualRolloutStrategy struct{}
 type QueueStatistics struct {
```

Comment:
Delete this whole struct; task queue stats update too frequently to be included in worker status.